### PR TITLE
fk/Up pylinter version 

### PR DESCRIPTION
**Pull request by FyodorK**
This PR has been created for test propose only please ignore it.

### CODE REVIEW

**Comment by FyodorK** [comment]
Jenkins please build only Pylint build

**Comment by FyodorK** [comment]
Jenkins please stop all builds

**Comment by FyodorK** [comment]
Additional logs against Pylint 2.10.0 
paased http://213.221.44.203/job/nncf/job/precommit/job/manual_NNCF/20/
failed (added long line to a py file)  http://213.221.44.203/job/nncf/job/precommit/job/manual_NNCF/19/

File: .pylintrc [hunk 1]
```diff
@@ -19,7 +19,24 @@ disable = arguments-differ,
...
+          saved_coeffs_in_file[key],
+          unsubscriptable-object,
+          node_key_list,
+          unsupported-membership-test,
+          deprecated-decorator,
+          consider-using-dict-items,
+          use-list-literal,
+          use-dict-literal,
+          use-a-generator,
+          use-maxsplit-arg
```
Comment (vshampor) [inline .pylintrc:39]: Shouldn't we instead fix the corresponding places in code? @alexsu52 is there anyone up for this right now?

Comment (FyodorK) [reply .pylintrc:39]: This is will be the right thing. But  ofc it is your call.

**Comment by FyodorK** [comment]
This PR is obsolete since #937 has been merged. Closed.

**Pull request closed without merging**